main.cpp

```diff
@@ -19,7 +19,6 @@
 **/
 
 #include <bits/stdc++.h>
-#include "playerData.h"
```

Comment:
This include does not need to be removed. We'll be using the header file later on in the project, in main.cpp